client/src/components/UserPost/index.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Userpost = props => {
+  const titles = props.post.map(title => <Link to={"/post/" + title.id}>{title.title}</Link>);
```

Comment:
this Link should also be wrapped in `<li>` with key

Reply:
i just removed user post and made it so that bio component can use suggestions to display the users posts